concepts/variables.coral

```diff
@@ -1,9 +1,38 @@
 /*
 Contains example variables and their contexts.
+
+Default inffered types:
 */
 
 // The following two variable declarations are exacly the same.
-let example = "Hello, world!";
-let String example2 = "Hello, world!";
+let onions = "Onions";
+let String beef = "Beef";
```

Comment:
Since we're allowing both of them, I think we should change it to:
```suggestion
let onions = "Onions";
let beef: String = "Beef";
```
That's just my personal opinion. Currently, it just looks weird to me.
If you agree with this change, this needs to be changed everywhere, keep that in mind.
Otherwise, if you disagree, please let me know why so I can think of another solution than this.

Reply:
Your syntax makes more sense in my eyes as the type can be optional.

Reply:
For type definition in functions, it would make sense that these also get defined in this syntax.  
So it would be something like this right:  
\`\`\`go  
func sample(param: String) {  
    ....  
}  
\`\`\`